nuxeo-runtime/nuxeo-runtime-migration/src/main/java/org/nuxeo/runtime/migration/MigrationServiceImpl.java

```diff
@@ -296,18 +297,26 @@ public boolean isShutdownRequested() {
 
         protected final MigrationContext migrationContext;
 
-        public MigratorWithContext(Consumer<MigrationContext> migrator, ProgressReporter progressReporter) {
+        protected final BiConsumer<MigrationContext, Throwable> cleaner;
```

Comment:
Is it interesting to rename `cleaner` by `finisher` ? As the code in this class around 485L does more than cleaning.

Reply:
I did additional renaming